middleware/kubernetes/setup.go

```diff
@@ -196,7 +229,33 @@ func kubernetesParse(c *caddy.Controller) (*Kubernetes, error) {
 	return nil, errors.New("kubernetes setup called without keyword 'kubernetes' in Corefile")
 }
 
+func getSearchPathFromResolvConf(filename string) ([]string, error) {
```

Comment:
I think you could use https://github.com/miekg/dns/blob/master/clientconfig.go#L22 to do this parsing.